src/OpenLoco/Localisation/StringManager.h

```diff
@@ -110,4 +110,5 @@ namespace OpenLoco::StringManager
 {
     const char* getString(string_id id);
     char* formatString(char* buffer, string_id id, const void* args = nullptr);
+    char* formatString(char* buffer, size_t bufferLen, string_id id, const void* args = nullptr);
```

Comment:
Both gcc and clang complain about `size_t` not having been declared:
```
../src/OpenLoco/Localisation/StringManager.h:113:38: error: ‘size_t’ has not been declared
  113 |     char* formatString(char* buffer, size_t bufferLen, string_id id, const void* args = nullptr);
      |                                      ^~~~~~
```
I'm a little puzzled as to why this happens here in particular, but perhaps `<cstddef>` should just be included to resolve it.

Reply:
maybe just `std::size_t` would fix it.